rfcs/20180731-dockerfile-assembler.md

```diff
@@ -0,0 +1,165 @@
+# TensorFlow Dockerfile Assembler
+
+| Status        | Proposed       |
+:-------------- |:---------------------------------------------------- |
+| **ID**        | <this will be allocated on approval>                 |
```

Comment:
This ID line can be deleted, it's left in the template in error. The ID is actually the filename of the RFC. I've fixed the template now.

Reply:
Fixed, thanks!